homeassistant/config_entries.py

```diff
@@ -831,6 +841,10 @@ async def async_unload(
 
     async def async_remove(self, hass: HomeAssistant) -> None:
         """Invoke remove callback on component."""
+        old_modified_at = self.modified_at
+        object.__setattr__(self, "modified_at", utcnow())
+        self.clear_cache()
```

Comment:
Not sure if we need it, as the only usecase for it is to update the modified_at date for the removed event